calico/felix/test/test_fetcd_api.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-

```

Comment:
We usually keep all the tests for foo.py in test_foo.py, any reason not to combine this with the above?